Makefile.nt

```diff
@@ -18,8 +18,62 @@ include Makefile.shared
 defaultentry:
 	@echo "Please refer to the installation instructions in file README.win32.adoc."
 
+REQUIRE_FLEXDLL:=$(if $(shell which flexlink 2>/dev/null),,$(FLEXDLL_SRC)/flexlink$(EXE))
+BOOT_FLEXLINK_CMD=$(if $(REQUIRE_FLEXDLL),FLEXLINK_CMD="../boot/ocamlrun ../$(FLEXDLL_SRC)/flexlink.exe")
+# Sys.command doesn't accept / in PATHs (because cmd doesn't)
```

Comment:
And do you insist on keeping the FLEXDLL_SRC variable? If relative paths (from the root of the project) could be hard-coded, this would avoid the problem as well.  Btw, one could probably share multiple occurrence of the redefinition for CAMOPT (to add OCAML_FLEXLINK) if one defined in each Makefile the relative path to the root of the project, no?